sdk/communication/communication-phone-numbers/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.2.0-beta.4 (2022-05-05)
```

Comment:
Isn't this supposed to be alpha if it's a private preview?

Reply:
Not really, alpha designation on packages in JS are used for labeling current master after merge. Here we are adding a functionality, so it seems it makes sense to have it as beta. (alpha will be a package automatically generated from this version after merge).